Minimum Character check for Search Component (#736)
ONSdigital/design-system: src/components/search/_macro-options.md

| width                      | string                               | false    | Numeric value for [input width constraint](/components/input/#width-constrained) defaults to '6'                                                      |
| label                      | `Label` [_(ref)_](/components/label) | false    | Settings for the input label. `for` will automatically be set to match the input id                                                                   |
| charcheck                  | number                               | false    | The maximum amount of characters a user should type in                                                                                                |

> | charcheckMinOnly           | boolean                              | false    | Required if `charcheck` is supplied. Will only check that the minimum amount of characters are entered, with no restriction on the maximum allowed characters         |

Reviewer: ```suggestion
| charcheckMinOnly           | boolean                              | false    |  If set to true will only check that the minimum amount of characters are entered, with no restriction on the maximum allowed characters         |
```
